### PR TITLE
v1.1: tests: backport changes to collect snapshot restore perf baselines 

### PR DESCRIPTION
**Pull request by kalyazin**
## Changes

This brings changes in test framework and baseline collection script from main in order to be able to collect performance baselines for snapshot restore.
The change includes update in the baseline config schema and baseline config files themselves.

## Reason

It was impossible to collect snapshot restore performance baselines for v1.1 in order to compare them with the main within the same environment (e.g., same testing instances). 

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

- [x] All commits in this PR are signed (`git commit -s`).
- [x] If a specific issue led to this PR, this PR closes the issue.
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- [x] New `unsafe` code is documented.
- [x] API changes follow the [Runbook for Firecracker API changes][2].
- [x] User-facing changes are mentioned in `CHANGELOG.md`.
- [x] All added/changed functionality is tested.

---

- [ ] This functionality can be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md

### CODE REVIEW

**Review by pb8o — DISMISSED** [review]

**Review by aghecenco — APPROVED** [review]

File: tests/integration_tests/performance/test_versioned_serialization_benchmark.py [hunk 1]
```diff
@@ -6,14 +6,14 @@
 import logging
 import json
 import shutil
+import platform
 
-import pytest
```
[the remark this answers is not in the document]
Comment (kalyazin) [reply tests/integration_tests/performance/test_versioned_serialization_benchmark.py:10]: unfortunately yes. Due to the 4.14 kernel update the old baselines don't look relevant any more...

[the remark this answers is not in the document]
Comment (kalyazin) [reply tests/integration_tests/performance/test_versioned_serialization_benchmark.py:10]: I was planning to explain all the additional changes once I get this through CI.

[the remark this answers is not in the document]
Comment (pb8o) [reply tests/integration_tests/performance/test_versioned_serialization_benchmark.py:10]: ah sorry I see it's still Draft.

[the remark this answers is not in the document]
Comment (kalyazin) [reply tests/integration_tests/performance/test_versioned_serialization_benchmark.py:10]: No, it's me to say sorry as it wasn't a draft when you looked at it :)

**Review by pb8o — APPROVED** [review]

**Review by zulinx86 — APPROVED** [review]

**Pull request merged**